src/invidious.cr

```diff
@@ -101,6 +102,10 @@ Kemal.config.extra_options do |parser|
     puts SOFTWARE.to_pretty_json
     exit
   end
+  parser.on("--migrate", "Run any migrations") do
+    Invidious::Database::Migrator.new(PG_DB).migrate
+    exit
+  end
```

Comment:
This was a good idea, only issue is that it really messes with my flow of using docker-compose

Reply:
Oh, right. We should probably make a wrapper script for that?

Also, we probably shouldn't `exit` if migrations are run. Passing `--migrate` would always bump the DB no matter what, without breaking the execution process. And we could make that the default on Docker, so owners of small instances aren't too surprised.

Reply:
So I'm concerned that we're going to roll out a breaking change to everyone with this PR. Everyone will update as normal and then find out that they can't run the service and will have to update their command, and I'm not sure the best way to roll that out.

What I'm going to do is leave this as it is, and remove the pending migration check. That way we can merge this change, it won't break anyone, and people can test out the migrations process. Then we can make a follow-up PR with the implementation that requires the migrations.

Reply:
I am struggling to run these CLI arguments, when I type `sudo docker exec -it invidious-invidious-1 invidious -v` I get

> OCI runtime exec failed: exec failed: unable to start container process: exec: "invidious": executable file not found in $PATH: unknown